app/controllers/users/registrations_controller.rb

```diff
@@ -18,8 +23,53 @@ def create
     else
       redirect_to root_path, notice: 'User cannot be added'
     end
+  end
+
+  def new
+    @user = User.new   
+    @users = User.all
+     
+  end
 
+  def show  
+    @user = User.find params[:id]
   end
+ 
+  def destroy
+    @user = User.find params[:format]
+    if @user.destroy  
+      redirect_to '/users', notice: 'User was successfully destroyed'
+    end
```

Comment:
This might error in the case where there is an error and the user couldn't be destroyed for some reason.  Since there is no else to handle that case it will have no where to redirect to and I think that will throw an error.

Reply:
Yes this is the error I fixed in the last admin branch. I know there is a bug also for warren we never used :format instead of :id I think I learned a lot implemented the role base users. Also, the messages that were disabled for the demo made me look for John's help and he showed me how to handle the errors with pundit. You can see the progress from all my branches from admin1 until admin14 how was my progress to figure out things with devise, pundit and users at the same time. Please check warren cause in the users controllers there are two actions that handle :format instead of the :id.